src/components/VImg/VImg.vue

```diff
@@ -1,20 +1,25 @@
 <template>
   <div :class="['vts-img', classes.root]">
     <div
-      v-if="dataUrl"
       ref="placeholder"
+      v-if="placeholder || dataUrl"
       :class="['vts-img__placeholder', classes.placeholder]"
       :style="{ background }"
     >
-      <img :src="placeholder || dataUrl" alt="" v-bind="$attrs" />
+      <img
+        :src="placeholder || dataUrl"
+        v-bind="$attrs"
+        :class="classes.img"
```

Comment:
@toyflish Were you having issues with missing classes here or something not working as expected? Trying to understand how you are using the component.